MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -24,11 +15,17 @@
 using MonoDevelop.MSBuild.SdkResolution;
 using MonoDevelop.Xml.Dom;
 using MonoDevelop.Xml.Editor.Completion;
+using static MonoDevelop.MSBuild.Language.ExpressionCompletion;
 using MonoDevelop.Xml.Parser;
 
 using ProjectFileTools.NuGetSearch.Feeds;
 
-using static MonoDevelop.MSBuild.Language.ExpressionCompletion;
+using System.Collections.Generic;
```

Comment:
System directives first, use the right click "remove and sort usings" command and it will do the right thing

Reply:
Sorted usings